tools/ci/benchmark-travis-pr.sh

```diff
@@ -23,20 +26,35 @@ run_asv () {
     pip install -e .
     git show --no-patch --format="%H (%s)"
     configure_asv
-    asv run -E existing --set-commit-hash $(git rev-parse HEAD)
+    asv run -E existing --set-commit-hash "$(git rev-parse HEAD)"
```

Comment:
Quotes aren't needed in this case.  (If you grep through git's source code you'll see that they usually do not surround `$(git rev-parse HEAD)` with quotes.)  But if it eases your mind or you like the consistency, it of course doesn't hurt.

Reply:
I succumb to pleasures of shellcheck. If there is no harm - I just obey

Reply:
Fair enough.